docker-compose-mac.yml

```diff
@@ -120,6 +120,17 @@ services:
       start_period: 4s
       interval: 5s
 
+  storybook:
```

Comment:
Since this isn't a production service, would it potentially make more sense to put it in another docker-compose file?

Reply:
Indeed. It's fixed.